README.md

```diff
@@ -5,13 +5,13 @@ This bookdown book is a *work in progress*. We'll update this `README` and the r
 
 # Requirements to preview the bookdown locally 
 
-## OMDb API Key
+## OMDb API key
```

Comment:
Let's move this section to the very bottom, to better reflect priorities. In #27, I'm asking for more, but let's not muddy the waters here.

Reply:
Much of what I say in #27 is now here, i.e. making the site render-able without an OMDb key.